frontend/app-development/features/appPublish/components/DeploymentEnvironmentStatus.tsx

```diff
@@ -88,7 +88,11 @@ export const DeploymentEnvironmentStatus = ({
             version: kubernetesDeployment.version,
           }}
           components={{
-            a: <Link href={urlToApp}> </Link>,
+            a: (
+              <Link href={urlToApp} rel='noopener noreferrer' target='_blank'>
```

Comment:
I suggest adding a text to inform users that this link will open a new tab/window, as it can be disorienting for those using screen readers:
- https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML#external_links_and_linking_to_non-html_resources

The best solution might be to show an icon and display the text only to screen reader users as suggested here:
- https://theadminbar.com/accessibility-weekly/opening-links-in-new-tabs-or-not/